src/components/ui/spinner.tsx

```
@@ -0,0 +1,9 @@
+import React from 'react'
+
+type Props = {}
+
+export default function Spinner({}: Props) {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)